i18n/en-US.properties

```diff
@@ -1592,18 +1592,14 @@ boxui.unifiedShare.settingsButtonLabel = Open shared link settings popup
 boxui.unifiedShare.sharedLinkDisabledTooltipCopy = Create and copy link for sharing
 # Tooltip describing when this shared link will expire. {expiration, date, long} is the formatted date
 boxui.unifiedShare.sharedLinkExpirationTooltip = This link will expire and be inaccessible on {expiration, date, long}.
-# Label for a shared link permission to show for editable box notes
+# Label for a shared link permission to show for an editable box note / file
 boxui.unifiedShare.sharedLinkPermissionsEdit = Can edit
 # Text to use in the tooltip when presenting an editable Box Note (DO NOT TRANSLATE "Box Notes")
 boxui.unifiedShare.sharedLinkPermissionsEditTooltip = This permission can only be changed in Box Notes
 # Label for a shared link permission level
 boxui.unifiedShare.sharedLinkPermissionsViewDownload = Can view and download
-# Description for Can view and download option
-boxui.unifiedShare.sharedLinkPermissionsViewDownloadDescription = Users can view and download
```

Comment:
Removed these messages because in the latest designs, the description is somewhat redundant.